content/22.vaccines.md

```diff
@@ -3,72 +3,66 @@
-<!--To do: tie into modular-->
-In light of the challenges of vaccine development, rapidly adaptable vaccine technologies that could be rapidly applied to new viruses became particularly appealing [@doi:10.1038/nrmicro3143; @doi:10.1016/j.vaccine.2017.02.069].
-Ideally, such an approach would facilitate the development of vaccines that could be adjusted to fit a novel viral threat at the minimum necessary degree and therefore could enter trials quickly in response to a new epidemic.
-<!--To Do: add https://doi.org/10.1016/j.vaccine.2017.02.069-->
+Over the past 150 years, several new approaches to vaccine development have emerged [@doi:10.1073/pnas.1400472111].
+Today, the requirements for developing and deploying a vaccine are complex and often require coordination between government, industry, academia, and philanthropic entities [@doi:10.1126/science.abc5312].
+Flu-like illnesses caused by viruses that follow an annual pattern are a major target of vaccine development programs.
+However, vaccine development has historically been slow.
+The past 20 years have seen several previously unknown viruses emerge and rise rapidly to pose a global threat, challenging vaccine developers to explore approaches that would facilitate a rapid response to novel viruses.
+
+Emergent viral threats of the 21st century include severe acute respiratory syndrome (SARS), “swine flu” (H1N1), Middle East respiratory syndrome (MERS), Ebola virus disease (EVD), and now COVID-19, all of which have underscored the importance of a rapid global response to a new infectious virus.
+Because vaccines fail to provide immediate prophylactic protection or treatment of ongoing infections, their application to most of these epidemics has been limited [@doi:10.3390/v10120680].
+One of the more successful recent vaccine development programs was for H1N1 influenza.
+This program benefited from the strong existing infrastructure for influenza vaccines along with the fact that regulatory agencies had determined that vaccines produced using egg- and cell-based platforms could be licensed under the regulations used for a strain change [@doi:10.1056/NEJMp2005630].
+Although a monovalent H1N1 vaccine was not available before the pandemic peaked in the United States and Europe, it became available soon afterward as a stand-alone vaccine that was eventually incorporated into commercially available seasonal influenza vaccines [@doi:10.1056/NEJMp2005630].
+Critiques of the production and distribution of the H1N1 vaccine have stressed the need for alternative development-and-manufacturing platforms that can be readily adapted to new pathogens. 
+
+Rapidly adaptable vaccine technologies would facilitate the development of vaccines that could be adjusted to fit a novel viral threat at the minimum necessary degree and therefore could enter trials quickly in response to a new virus of concern.
```

Comment:
```suggestion
Modular vaccine technologies are appealing because a vaccine that required only minor adjustments to fit a novel viral threat could enter trials quickly in response to a new pathogen of concern.
```

I agree, it's awkward! What about this instead?